README.md

```diff
@@ -24,19 +24,19 @@ const { TextDecoder, TextEncoder } = require('text-encoding');  // node, IE11 an
 SignatureProvider holds private keys and is responsible for signing transactions
 ```js
 const defaultPrivateKey = "5JtUScZK2XEp3g9gh7F8bwtPTRAkASmNrrftmx4AxDKD5K4zDnr"; // useraaaaaaaa
-const signatureProvider = new eosjs2.SignatureProvider([defaultPrivateKey]);
+const signatureProvider = new eosjs.SignatureProvider([defaultPrivateKey]);
```

Comment:
Have these changes also happened in the code?

Reply:
They haven't